backend/ops_api/tests/ops/features/validate_draft_budget_lines.feature

```diff
@@ -91,14 +91,6 @@ Feature: Validate "Draft" Budget Lines
 
     Then I should get an error message that the BLI's Agreement must have a Project Officer
 
-  Scenario: Valid Team Members
```

Comment:
learned in today's demo that Team members are not required to be sent for review